src/panels/lovelace/elements/hui-state-label-element.ts

```diff
@@ -57,7 +57,7 @@ class HuiStateLabelElement extends LitElement implements LovelaceElement {
 
     if (
       this._config.attribute &&
-      !stateObj.attributes[this._config.attribute]
+      stateObj.attributes[this._config.attribute] === undefined
```

Comment:
```suggestion
      !this._config.attribute in stateObj.attributes
```

Reply:
Typescript doesn't seem to like this change, I'm guessing this is because the type of attribute is `string?` rather than `string`.  I think for this to work we'd first need to check if it's undefined, then check if the attribute is in the `stateObj`.